packages/nimble-components/src/table-column/anchor/index.ts

```diff
@@ -158,6 +174,13 @@ export class TableColumnAnchor extends mixinGroupableColumnAPI(
             placeholder: this.placeholder
         };
     }
+
+    private updateOperandDataRecordFieldName(): void {
```

Comment:
this is doing more than the name suggests, what about  `updateColumnInternalsSortConfig`?
Same applies to text column